compiler/rustc_mir_build/src/thir/pattern/_match.rs

```diff
@@ -79,19 +79,21 @@
 //!
 //! For example, say we have the following:
 //! ```
-//!     // x: (Option<bool>, Result<()>)
-//!     match x {
-//!         (Some(true), _) => {}
-//!         (None, Err(())) => {}
-//!         (None, Err(_)) => {}
-//!     }
+//! // x: (Option<bool>, Result<()>)
+//! match x {
+//!     (Some(true), _) => {}
+//!     (None, Err(())) => {}
+//!     (None, Err(_)) => {}
+//! }
 //! ```
 //! Here, the matrix `P` starts as:
+//! ```
 //! [
 //!     [(Some(true), _)],
 //!     [(None, Err(()))],
 //!     [(None, Err(_))],
 //! ]
+//! ```
```

Comment:
the general comment style is to put an empty line between code fences and the stuff around it, that is

```
//! ```
//!
//! We can tell it's
```

doing this for all of the code fences in this PR would be appreciated.

Reply:
I actually prefer that, but all the other code snippets I saw in this page don’t have blank lines, so I was going with that style. I’ll add some blank lines.

Reply:
Done.